lib/helper/Mochawesome.js

```diff
@@ -67,6 +68,7 @@ class Mochawesome extends Helper {
   }
 
   addMochawesomeContext(context) {
+    if (currentTest = "") currentTest = { test: currentSuite.ctx.test }
```

Comment:
Hmm, shouldn't this be `currentTest ==`?  Or am I missing something? :)

Reply:
Yep will fix this)